documentation/CLI_usage.md

```diff
@@ -0,0 +1,195 @@
+# CLI Usage
+
+---
+## npd
+
+### Description
+napari plugin devtools scans the current python environment to find out details about napari plugin information.
+
+### Usage
+```
+npd [COMMANDS] [ARGS...]
+```
+
+### Commands
+|Command        |Description                                          |
+|---------------|-----------------------------------------------------|
+|validate       |validate the given package is correctly setup        |
+|discover       |discover all plugins under current python environment|
+
+
+## npd validate
+
+### Description
+Validate package(s) within current python environment, report PASSED or FAILED for each check, 
+only return 1 when all checks passed.
+
+### Usage
+```
+npd validate [Options] <package> [<packages>...]
+```
+
+### Options
+|name, shorthand    |Description                     |
+|-------------------|--------------------------------|
+|--verbose, -v      |Show more detailed output       |
+
+### Examples
+
+validate single package
+```
+$ pip install napari-svg
+$ npd validate napari-svg
+----------------------------------------------------------------
+Scanning current python environment: /Library/Frameworks/Python.framework/Versions/3.8
+----------------------------------------------------------------
+Validating package napari-svg
+* Classifier check: FAILED
+* Entrypoint check - syntax: PASSED
+* Entrypoint check - plugin registration: PASSED
+* Entrypoint check - registered modules exist: PASSED
+* Hooks registration check - svg: PASSED
+----------------------------------------------------------------
+Error: 'Framework :: napari' does not exist for napari-svg's 12 known classifier(s)
```

Comment:
Error location is good, but I still could be confused on what to do to resolve this. Part of the error message is a little confusing with "12 known classifier(s)". Could it also point me to a line in the code or even just a file name where I should add this classifier?